packages/client-core/src/systems/AvatarUISystem.tsx

```diff
@@ -94,7 +94,7 @@ export const renderAvatarContextMenu = (userId: UserID, contextMenuEntity: Entit
   contextMenuXRUI.quaternion.copy(cameraTransform.rotation)
 }
 
-const userQuery = defineQuery([AvatarComponent, TransformComponent, NetworkObjectComponent, Not(NetworkObjectOwnedTag)])
+const userQuery = defineQuery([AvatarComponent, TransformComponent, NetworkObjectComponent]) //, Not(NetworkObjectOwnedTag)])
```

Comment:
the `Not(NetworkObjectOwnedTag)` is to exclude your own avatar from having a nameplate, is this handled somewhere else now?

Reply:
ah I see, in the reactor. the query would make more sense.